doc/getting-started/overview_dev.rst

```diff
@@ -3,19 +3,22 @@
 Configuration and Development Overview
 ######################################
 
-This overview is for developers who are new or relatively new to ACRN. It will
-help you get familiar with ACRN basics: ACRN components and general process for
-building an ACRN hypervisor.
+This overview is for developers who are new or relatively new to ACRN and are
+responsible for configuring and building ACRN hypervisors. It will introduce you
```

Comment:
Technically, we're configuring and building the hypervisor and the VMs for the application.